app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] It would be nice to use the `after_save` to call the `#update_comments_counter` method every time a new comment has been saved to database, then make the `update_comment_counter` method private, just like in the post model, check the post model to see the suggestion 👍🏿

Reply:
This change has been implemented well